docs/user-guide/ui.md

```diff
@@ -2,48 +2,82 @@
+
+* TOC
+{:toc}
 
-## Using the Dashboard
+## Accessing the Dashboard
 
-The Dashboard can be used to get an overview of applications running on the cluster, and to provide information on any errors that have occurred. You can also inspect your replication controllers and corresponding services, change the number of replicated Pods, and deploy new applications using a deploy wizard.
+Navigate in your Browser to the following URL:
+```
+https://<kubernetes-master>/ui
+```
+This redirects to the following URL:
+```
+https://<kubernetes-master>/api/v1/proxy/namespaces/kube-system/services/kubernetes-dashboard
+```
+The Dashboard UI lives in the `kube-system` [namespace](/docs/admin/namespaces/), but shows all resources from all namespaces in your environment.
+
+If you find that you are not able to access the Dashboard, you can install and open the latest stable release by running the following command:
+
+```
+kubcetl create -f https://rawgit.com/kubernetes/dashboard/master/src/deploy/kubernetes-dashboard.yaml
+```
+
+Then, navigate to
+
+```
+https://<kubernetes-master>/ui
+```
+
+In case you have to provide a password, use the following command to find it out:
 
-When accessing the Dashboard on an empty cluster for the first time, the Welcome page is displayed. This page contains a link to this document as well as a button to deploy your first application. In addition, you can view which system applications are running by default in the `kube-system` [namespace](/docs/admin/namespaces/) of your cluster, for example monitoring applications such as Heapster.
+```
+kubectl config view
+```
+
+## Welcome page
+
+When accessing the Dashboard on an empty cluster for the first time, the Welcome page is displayed. This page contains a link to this document as well as a button to deploy your first application. In addition, you can view which system applications are running by **default** in the `kube-system` [namespace](/docs/admin/namespaces/) of your cluster, for example monitoring applications such as Heapster.
 
 ![Kubernetes Dashboard welcome page](/images/docs/ui-dashboard-zerostate.png)
 
-### Deploying applications
+## Deploying containerized applications
 
-The Dashboard lets you create and deploy a containerized application as a Replication Controller with a simple wizard:
-
-![Kubernetes Dashboard deploy form](/images/docs/ui-dashboard-deploy-simple.png)
-
-#### Specifying application details
+The Dashboard lets you create and deploy a containerized application as a Replication Controller and corresponding Service with a simple wizard. You can either manually specify application details, or upload a YAML or JSON file containing the required information.
+
+To access the deploy wizard from the Welcome page, click the respective button. To access the wizard at a later point in time, click the plus (+) sign in the upper right corner of any  page.
```

Comment:
Both buttons still lead to the same Deploy page, just with one or the other options pre-selected.